src-django/authentication/tests/test_confirm_email.py

```diff
@@ -0,0 +1,53 @@
+from django.test import TestCase, Client
+from nose.tools import assert_equals, assert_true, assert_false, assert_raises
+from authentication.models import EmailConfirmationKey
+from utils import factories
+from rest_framework import status
+import datetime
+import json
+
+
+class ConfirmEmailTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+
+    def test_valid_key(self):
+        email_confirmation_key = factories.EmailConfirmationKeyFactory()
+        response = self.client.get('/auth/confirm_email/{0}'.format(email_confirmation_key.key))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
oh weird, I was following the convention in [src-django/authentication/tests/test_signup.py](https://github.com/SanaMobile/sana.protocol_builder/blob/master/src-django/authentication/tests/test_signup.py), but I agree it should be `assert_equals`